pkg/crc/machine/bundle/metadata_test.go

```diff
@@ -188,24 +189,31 @@ func TestGetBundleType(t *testing.T) {
 }
 
 func TestVerifiedHash(t *testing.T) {
-	sha256sum, err := getVerifiedHash("https://developers.redhat.com/content-gateway/file/pub/openshift-v4/clients/crc/bundles/openshift/4.13.0/sha256sum.txt.sig", "crc_libvirt_4.13.0_amd64.crcbundle")
+	absPath, err := filepath.Abs(filepath.Join("testdata", "sha256sum_correct_4.13.0.txt.sig"))
+	require.NoError(t, err)
+	sha256sum, err := getVerifiedHash(fmt.Sprintf("file:///%s", absPath), "crc_libvirt_4.13.0_amd64.crcbundle")
 	require.NoError(t, err)
 	require.Equal(t, "6aad57019aaab95b670378f569b3f4a16398da0358dd1057996453a8d6d92212", sha256sum)
 
 	// sha256sum.txt is unsigned
-	_, err = getVerifiedHash("https://developers.redhat.com/content-gateway/file/pub/openshift-v4/clients/crc/bundles/openshift/4.13.0/sha256sum.txt", "crc_libvirt_4.13.0_amd64.crcbundle")
+	absPath, err = filepath.Abs(filepath.Join("testdata", "sha256sum.txt"))
+	require.NoError(t, err)
+	_, err = getVerifiedHash(fmt.Sprintf("file:///%s", absPath), "crc_libvirt_4.13.0_amd64.crcbundle")
 	require.Error(t, err)
 
 	// sha256sum.txt.sig does not contain a sha256sum for fake.crcbundle
-	_, err = getVerifiedHash("https://developers.redhat.com/content-gateway/file/pub/openshift-v4/clients/crc/bundles/openshift/4.13.0/sha256sum.txt.sig", "fake.crcbundle")
+	absPath, err = filepath.Abs(filepath.Join("testdata", "sha256sum_correct_4.13.0.txt.sig"))
+	require.NoError(t, err)
+	_, err = getVerifiedHash(fmt.Sprintf("file:///%s", absPath), "fake.crcbundle")
 	require.Error(t, err)
 
 	// 4.10.1 bundles did not come with a sha256sum.txt.sig file
 	_, err = getVerifiedHash("https://developers.redhat.com/content-gateway/file/pub/openshift-v4/clients/crc/bundles/openshift/4.10.1/sha256sum.txt.sig", "crc_libvirt_4.10.1_amd64.crcbundle")
```

Comment:
I am not sure what to do there because this is anyway return `404` since file is not found and we are just saying if there is err (no matter what type) then we are good.

Reply:
The goal of this test was to make sure we don't return false positives when the file is missing (ie don't return a hash by mistake). If you replace this with a file, it's going to be quite different than testing with http.
You can keep it anyway with a file, or remove it, no strong feeling.

Reply:
Removed this one.